package.json

```diff
@@ -290,6 +290,10 @@
 						"maxResults": {
 							"type": "number",
 							"description": "The maximum number of results to return. Do not use this unless necessary, it can slow things down. By default, only some matches are returned. If you use this and don't see what you're looking for, you can try again with a more specific query or a larger maxResults."
+						},
+						"includeIgnoredFiles": {
+							"type": "boolean",
+							"description": "Whether to include files that would normally be ignored according to .gitignore, other ignore files and `files.exclude` and `search.exclude` settings."
```

Comment:
Include a message like `Warning: using this may cause the search to be slower. Only set it when you want to search in ignored folders like node_modules or build outputs.`

Reply:
Hmm, sometimes it did pass in `includeIgnoredFiles = true` but today quite often it doesn't pass in like that. Let me try some tweaks.

Reply:
This would be a great one to have a vscbench eval for